openmc/tallies.py

```diff
@@ -1729,10 +1749,14 @@ def swap_filters(self, filter1, filter2):
         filter2 : Filter
             The filter to swap with filter1
 
+        inline : bool, optional
```

Comment:
Looks like we have spaces between our parameters in this docstring which is weird and inconsistent with the rest of the API's docstrings. Can you remove the blank lines between parameters here?